### PR TITLE
Catalogue Entries V1

### PR DESCRIPTION
**Pull request by klaasjared**
Adds Cataloguer lore that needed to be done for a while now...

Lore that was added
-Deathclaw
-Fennec
-Giant Frog
-Snake
-Giant Snake
-Giant Rat
-Sect Queen
-Sect Drone (also added better dead sprite)
-Mimic (also modified which .dmi that this sprite points to)

This also adds a new cataloguer sprite.

### CODE REVIEW

**Review by Novacat — CHANGES_REQUESTED** [review]
Wait these guys had cataloguer entries?

oh.

Someone forgot to add:
catalogue_data = list(/datum/category_item/catalogue/fauna/giant_spider/guard_spider)
(as an example)

To all the virgo voremobs vars. Can you do that please?

**Review by Novacat — APPROVED** [review]

**Review by Novacat — DISMISSED** [review]
Actually, I would like the spaces on the start of new lines to be converted into tabs instead

File: code/modules/mob/living/simple_mob/subtypes/vore/rat.dm [hunk 1]
```diff
@@ -1,14 +1,28 @@
...
+/datum/category_item/catalogue/fauna/rat
 	name = "Creature - Rat"
-	desc = "A massive rat, some sort of mutated descendant of normal Earth rats. These ones seem particularly hungry, \
-	and are able to pounce and stun their targets - presumably to eat them. Their bodies are long and greyfurred, \
-	with a pink nose and large teeth, just like their regular-sized counterparts."
+	desc = "Classification: Mus muscular\
+	<br><br>\
+	Rats are various medium-sized, long-tailed rodents. Species of rats are found throughout the order Rodentia, \
+	but stereotypical rats are found in the genus Rattus. This specific species of rat is a mutated decendant from lab rats.\
+	It is unclear what experiment caused this species to grow to such an unnatural size, however it hasn't effected the rat's \
```
Comment (TheFurryFeline) [inline code/modules/mob/living/simple_mob/subtypes/vore/rat.dm:7]:
```suggestion
	It is unclear what experiment caused this species to grow to such an unnatural size, however it hasn't affected the rat's \
```

File: code/modules/mob/living/simple_mob/subtypes/vore/rat.dm [hunk 2]
```diff
@@ -1,14 +1,28 @@
...
+	general docile nature. When encounterd by human's or other species it generally ignores them unless provoked.\
```
Comment (TheFurryFeline) [inline code/modules/mob/living/simple_mob/subtypes/vore/rat.dm:8]:
```suggestion
	general docile nature. When encounterd by humans or other species it generally ignores them unless provoked.\
```

File: code/modules/mob/living/simple_mob/subtypes/vore/frog.dm [hunk 3]
```diff
@@ -1,7 +1,27 @@
+/datum/category_item/catalogue/fauna/frog
+	name = "Wildlife - Giant Frog"
+	desc = "Classification: Anura gigantus\
+	<br><br>\
+	A frog is any member of a diverse and largely carnivorous group of short-bodied, tailless amphibians composing \
+	the order Anura. This specific species - Anura gigantus - is a mutated form of Frogs, largly due to exposure to mutagen chemicals. \
+	These Giant Frogs are decendants from scientific frogs that were used for study during the great Sol Expansion Period. \
+	Modern day Giant Frogs have reveted to a more feral state compared to their original anncestors and are hostile \
```
Comment (TheFurryFeline) [inline code/modules/mob/living/simple_mob/subtypes/vore/frog.dm:8]:
```suggestion
	Modern day Giant Frogs have reverted to a more feral state compared to their original ancestors and are hostile \
```

File: code/modules/mob/living/simple_mob/subtypes/vore/frog.dm [hunk 4]
```diff
@@ -1,7 +1,27 @@
...
+	towards humans and other small wildlife - hunting them for food.\
+	<br>\
+	The particular breed of Frog that was originally used in the scientific experiments were known as explosive breeders.\
+	With explosive breeders, mature adult frogs arrive at breeding sites in response to certain trigger factors such as rainfall \
+	occurring in an arid area. In these frogs, mating and spawning take place promptly and the speed of larval growth is rapid in \
+	order to make use of the ephemeral pools before they dry up. Because of this, the Frog population is through the roof and has \
+	become a major issue for various colonies and stations.\
+	<br>\
+    These animals, are considered an invasive species, and thus hunters are encouraged to hunt them."
```
Comment (TheFurryFeline) [inline code/modules/mob/living/simple_mob/subtypes/vore/frog.dm:17]: Tab indent here, not spaces.

File: code/modules/mob/living/simple_mob/subtypes/animal/space/snake_vr.dm [hunk 5]
```diff
@@ -1,6 +1,24 @@
...
+	Snakes are elongated, limbless, carnivorous reptiles of the suborder Serpentes \
+	Like all other squamates, snakes are ectothermic, amniote vertebrates covered in overlapping scales. \
+	Many species of snakes have skulls with several more joints than their lizard ancestors, \
+	enabling them to swallow prey much larger than their heads with their highly mobile jaws. \
+	<br>\
+	This species of snake is nonvenomous and use their large bodies to primarily subdue their prey. \
+	Nonvenomous snakes either swallow prey alive or kill them by constriction - this is dependant on the prey. \
+	<br>\
+    These animals are a protected species and are not to be harmed, hunted, or captured by anyone. \
+    Severe consequences may result in violation of this order."
```
Comment (TheFurryFeline) [inline code/modules/mob/living/simple_mob/subtypes/animal/space/snake_vr.dm:14]: Tab indent here, not spaces.

File: code/modules/mob/living/simple_mob/subtypes/vore/deathclaw.dm [hunk 6]
```diff
@@ -1,13 +1,25 @@
-/datum/category_item/catalogue/fauna/deathclaw		//TODO: VIRGO_LORE_WRITING_WIP
+/datum/category_item/catalogue/fauna/deathclaw
 	name = "Creature - Deathclaw"
-	desc = "A massive beast, tall as three standard-size humans, with massive, terrifying claws, \
-	and dark, black fangs. It's entire body is yellowish, like sand, and it's skin is leathery and tough. \
-	It seems to have adapted to the harsh desert environment on Virgo 4, and makes it's home inside the caves."
+	desc = "Classification: Trioceros dominus\
+	<br><br>\
+	Originally the Deathclaw was a top secret genetics project run by a now defucnt defence company lost to time.\
+	The creatures are known to attack humans and other animals regularly to protect their territoy \
```
Comment (TheFurryFeline) [inline code/modules/mob/living/simple_mob/subtypes/vore/deathclaw.dm:6]:
```suggestion
	Originally the Deathclaw was a top secret genetics project run by a now defunct defence company lost to time.\
	The creatures are known to attack humans and other animals regularly to protect their territory \
```

File: code/modules/mob/living/simple_mob/subtypes/vore/snake.dm [hunk 7]
```diff
@@ -1,6 +1,29 @@
...
+	has grown to an unnatural size. Biologically this snake is no different than that of the common snake, \
+	but this species has been known to have increased hostility towards wildlife. Scientists are still studying \
+	this new species for any differences in behavior or biology beyond the increase in size. \
+	<br>\
+	This species of snake is nonvenomous and use their large bodies to primarily subdue their prey. \
+	Nonvenomous snakes either swallow prey alive or kill them by constriction - this is dependant on the prey. \
+	<br>\
+    This snake is extremely hostile to all wildlife and living beings and should be avoided at all costs. \
+    People who spot these creatures are urged to inform the nearest militant entity so that they can be \
+    dealt with in a professional manner."
```
Comment (TheFurryFeline) [inline code/modules/mob/living/simple_mob/subtypes/vore/snake.dm:19]: Tab indents

File: code/modules/mob/living/simple_mob/subtypes/vore/snake.dm [hunk 8]
```diff
@@ -1,6 +1,29 @@
+/datum/category_item/catalogue/fauna/giant_snake
+	name = "Creature - Giant Snake"
+	desc = "Classification: Serpentes gigantus\
+	<br><br>\
+	Snakes are elongated, limbless, carnivorous reptiles of the suborder Serpentes \
+	Like all other squamates, snakes are ectothermic, amniote vertebrates covered in overlapping scales. \
+	Many species of snakes have skulls with several more joints than their lizard ancestors, \
+	enabling them to swallow prey much larger than their heads with their highly mobile jaws. \
+	This particular species of snake has likely been mutated by the many phoron deposites in the region and \
```
Comment (TheFurryFeline) [inline code/modules/mob/living/simple_mob/subtypes/vore/snake.dm:9]:
```suggestion
	This particular species of snake has likely been mutated by the many phoron deposits in the region and \
```

**Review by TheFurryFeline — COMMENTED** [review]
Spelling fixes suggestions, but silly browser is preventing me from being able to do tabs that don't just go right to a button and out of the typing section, pfft.

File: code/modules/mob/living/simple_mob/subtypes/vore/sect_queen.dm [hunk 9]
```diff
@@ -2,20 +2,29 @@
...
-	smaller drones might be unable to acquire. The lack  of chitin on the underside of its abdomen is deliberate, \
-	as the flesh is very elastic and stretchable, allowing the queen to carry multiple large prey inside of its \
-	stomach with ease."
+	edges much like a Mantis, which it uses in both self-defense and during its hunts. Covering its body is a layer \
+	of thick and protective chitin, resilient enough to absorb most physical damage. \
+	<br>\
+	Though rarely seen, it is not uncommon for a queen to go out alone to search for potential new nesting grounds \
+	or perhaps it does so to seek bigger prey that its much smaller drones might be unable to acquire. \
+	Regardless of reason, it is cautioned against approaching a Sect Queen as their behaviour is wildly \
+	inconsistent. A Sect Queen can vary from hositle to docile depending on certain factors that scientists have \
```
Comment (TheFurryFeline) [inline code/modules/mob/living/simple_mob/subtypes/vore/sect_queen.dm:15]:
```suggestion
	inconsistent. A Sect Queen can vary from hostile to docile depending on certain factors that scientists have \
```

File: code/modules/mob/living/simple_mob/subtypes/vore/sect_drone.dm [hunk 10]
```diff
@@ -1,12 +1,27 @@
...
+	uses for increased mobility. Covering its body is a layer of light, thick, and protective chitin, resilient enough to absorb \
+	most physical damage while being light enough for the Sect Drone to hover. \
+	<br>\
+	It is not uncommon for a Sect Drone to go out alone to search for potential prey to bring back to the nest. \
+	Regardless of reason, it is cautioned against approaching a Sect Drone as, like their queen, their behaviour is wildly \
+	inconsistent. A Sect Drone can vary from hositle to docile depending on certain factors that scientists have \
+	yet to uncover. \
+	<br>\
+	The lack  of chitin on the underside of its abdomen is deliberate, as the flesh is very elastic and stretchable, \
+	allowing the drone to carry multiple large prey inside of its stomach with realitive ease."
```
Comment (TheFurryFeline) [inline code/modules/mob/living/simple_mob/subtypes/vore/sect_drone.dm:17]:
```suggestion
	inconsistent. A Sect Drone can vary from hostile to docile depending on certain factors that scientists have \
	yet to uncover. \
	<br>\
	The lack  of chitin on the underside of its abdomen is deliberate, as the flesh is very elastic and stretchable, \
	allowing the drone to carry multiple large prey inside of its stomach with relative ease."
```

**Review by TheFurryFeline — COMMENTED** [review]
Moar spelling fixies <3

**Comment by Novacat** [comment]
Going to wait for the Loremins to review this.

File: code/modules/mob/living/simple_mob/subtypes/vore/sect_drone.dm [hunk 11]
```diff
@@ -1,12 +1,28 @@
...
```
Comment (TheFurryFeline) [inline code/modules/mob/living/simple_mob/subtypes/vore/sect_drone.dm:17]:
```suggestion
	inconsistent. A Sect Drone can vary from hostile to docile depending on certain factors that scientists have \
	yet to uncover. \
	<br>\
	The lack  of chitin on the underside of its abdomen is deliberate, as the flesh is very elastic and stretchable, \
	allowing the drone to carry multiple large prey inside of its stomach with relative ease."
```

File: code/modules/mob/living/simple_mob/subtypes/vore/sect_queen.dm [hunk 12]
```diff
@@ -2,20 +2,30 @@
...
```
Comment (TheFurryFeline) [inline code/modules/mob/living/simple_mob/subtypes/vore/sect_queen.dm:15]:
```suggestion
	inconsistent. A Sect Queen can vary from hostile to docile depending on certain factors that scientists have \
```

**Review by TheFurryFeline — COMMENTED** [review]
Seems a commit undid the spell fixes here. Be sure to keep your branch synched.

**Review by Novacat — CHANGES_REQUESTED** [review]
The Loremins have taken a quick peek:

On the snakes :

These animals are a protected species and are not to be harmed, hunted, or captured by anyone. \
    Severe consequences may result in violation of this order."

No. We're not going to make people think getting a pet snake or something is going to be punished. I want that paragraph removed.


On the Fennecs :

Their presence on Virgo 4 spans back to when the colony was first established and now modern Virgo \
    Fennec are upwards of five times larger than their Sol cousins and have gained a larger appetite. \
    However, despite this they are passive and do not actively hunt as their bodies have grown accustomed \
    to less available food sources.\


Not... Really? I would make them more a case of convergent evolution who actually have little to do with Earth fennecs all that much.

These animals, despite their current Conservation status, are a protected species and are not to be \
    harmed, hunted, or captured by anyone. Severe consequences may result in violation of this order."

Again, no.


On the Giant Snakes :

This particular species of snake has likely been mutated by the many phoron deposits in the region and \
    has grown to an unnatural size. 

Not sure Virgo 4 maps have Phoron. Deliberate gene manipulation might be more the reason.


On the Deathclaw :

Originally the Deathclaw was a top secret genetics project run by a now defunct defence company lost to time.\
    The creatures are known to attack humans and other animals regularly to protect their territory \
    or to hunt for food. It is speculated that they escaped roughly around the time the Hegemony War \
    broke out as sighting of these beasts in the wild began around that time. \


They are a result of gene manipulation, but not from any company or human hand, but rather the ancestors of the Zorren. That lil' tidbit isn't known, but they do share the same genetic "signs" of Zorren.

**Review by Novacat — APPROVED** [review]

File: code/modules/mob/living/simple_mob/subtypes/vore/frog.dm [hunk 13]
```diff
@@ -1,7 +1,27 @@
+/datum/category_item/catalogue/fauna/frog
+	name = "Wildlife - Giant Frog"
+	desc = "Classification: Anura gigantus\
+	<br><br>\
+	A frog is any member of a diverse and largely carnivorous group of short-bodied, tailless amphibians composing \
+	the order Anura. This specific species - Anura gigantus - is a mutated form of Frogs, largly due to exposure to mutagen chemicals. \
+	These Giant Frogs are decendants from scientific frogs that were used for study during the great Sol Expansion Period. \
```
Comment (TheFurryFeline) [inline code/modules/mob/living/simple_mob/subtypes/vore/frog.dm:7]:
```suggestion
	the order Anura. This specific species - Anura gigantus - is a mutated form of Frogs, largely due to exposure to mutagen chemicals. \
	These Giant Frogs are descendants from scientific frogs that were used for study during the great Sol Expansion Period. \
```

File: code/modules/mob/living/simple_mob/subtypes/vore/fennec.dm [hunk 14]
```diff
@@ -1,15 +1,31 @@
...
+	desc = "Classification: Vulpes zerda maxima\
+	<br><br>\
+	The Fennec fox is a small crepuscular fox native to Earth in Sol that nearly went extinct in the 2030s.\
+	Through conservation efforts and the rise of space colonies, the Fennec was brought back from the brink \
+	and is now labeled as 'Least Concern'. During the great Sol Expansion Period, Fennec were brought with \
+	colonist as a means of companionship and as a ecosystem balance for desert worlds such as Virgo 4. \
+	Their presence on Virgo 4 is largely due to convergent evolution. While their genetics are closely \
+	related to their Sol counterparts, they are in fact a totally different species of fennec that have followed. \
+	a seperate evolutional path. Virgo Fennec are upwards of five times larger than their Sol cousins and consequently \
+	have a larger appetite. Thier diet mainly consists of whatever small creatures that they manage to scrounge from \
```
Comment (TheFurryFeline) [inline code/modules/mob/living/simple_mob/subtypes/vore/fennec.dm:12]:
```suggestion
	a separate evolutional path. Virgo Fennec are upwards of five times larger than their Sol cousins and consequently \
	have a larger appetite. Their diet mainly consists of whatever small creatures that they manage to scrounge from \
```

File: code/modules/mob/living/simple_mob/subtypes/vore/rat.dm [hunk 15]
```diff
@@ -1,14 +1,28 @@
...
 	name = "Creature - Rat"
-	desc = "A massive rat, some sort of mutated descendant of normal Earth rats. These ones seem particularly hungry, \
-	and are able to pounce and stun their targets - presumably to eat them. Their bodies are long and greyfurred, \
-	with a pink nose and large teeth, just like their regular-sized counterparts."
+	desc = "Classification: Mus muscular\
+	<br><br>\
+	Rats are various medium-sized, long-tailed rodents. Species of rats are found throughout the order Rodentia, \
+	but stereotypical rats are found in the genus Rattus. This specific species of rat is a mutated decendant from lab rats. \
+	It is unclear what experiment caused this species to grow to such an unnatural size, however it hasn't affected the rat's \
+	general docile nature. When encounterd by humans or other species it generally ignores them unless provoked.\
```
Comment (TheFurryFeline) [inline code/modules/mob/living/simple_mob/subtypes/vore/rat.dm:8]:
```suggestion
	but stereotypical rats are found in the genus Rattus. This specific species of rat is a mutated descendant from lab rats. \
	It is unclear what experiment caused this species to grow to such an unnatural size, however it hasn't affected the rat's \
	general docile nature. When encountered by humans or other species it generally ignores them unless provoked.\
```

File: code/modules/mob/living/simple_mob/subtypes/vore/rat.dm [hunk 16]
```diff
@@ -1,14 +1,28 @@
...
+	<br>\
+	Rats become sexually mature at age 6 weeks, but reach social maturity at about 5 to 6 months of age. \
+	The average lifespan of rats varies by species, but many only live about a year due to predation. \
+	However, due to the large nature of this particular species of rat, predation is usually not that much of an issue. \
+	This doesn't mean that there is an overpopulation, though, quite the opposite. Giant Rats are rare and this is usually \
+	due to small litter sizes and lack of proper food sources. Areas that one would typically see a Giant Rat is large garbage \
+	disposals or areas that have large amounts of live food (other rats, mice, etc.) such as maint tunnels. \
```
Comment (TheFurryFeline) [inline code/modules/mob/living/simple_mob/subtypes/vore/rat.dm:15]:
```suggestion
	disposals or areas that have large amounts of live food (other rats, mice, etc.) such as maintenance tunnels. \
```

**Review by TheFurryFeline — COMMENTED** [review]
One last round of spellchecks, yay.

**Pull request merged**